### DISCUSSION TITLE
Compatibility With Other Gitops Project

### DISCUSSION THREAD

**Discussion by infinitydon** [flux-iac/tofu-controller#1428] [General]
Hi,

Following up on our discussion from [this issue](https://github.com/flux-iac/tofu-controller/issues/284), I believe it would still be beneficial to make the controller GitOps-agnostic. This approach could broaden its usage at a larger scale, allowing it to work with various GitOps solutions apart from FluxCD.

Additionally, it seems that [Flamingo](https://github.com/flux-subsystem-argo/flamingo/issues/70) may not be actively maintained in the future, which reinforces the need for a more flexible and adaptable controller.

I will like to get opinions about this.

Thanks

**Comment by ilithanos** [comment]
I’m not completely sure what need that is to be honest.

We are tied to the flux way of thinking gitops, and if flamingo is no longer maintained I have a hard time seeing a good path towards what you are requesting.

To go support running without flux we would need to duplicate a lot of functionality such as the source controller which is used to handle git and oci sources etc.


You don’t need the entire fluxcd but you will need specific parts of it for this controller to work.

What are you requesting support for specifically?
